<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>40</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QLineEdit" name="lineEdit" >
      <property name="text" >
       <string>25</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Enter your age</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2" >
      <property name="text" >
       <string>&amp;quit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
